text/0249-v2-experiments.md

```diff
@@ -418,23 +675,6 @@ Disadvantages:
   other instances of using unstable code, working against stated goal #1.
 - Requires changes in JSII.
 
-### Force a naming convention for unstable code
```

Comment:
In my opinion, this is where proposal 6 should live, as it's not a full-fledged solution to the problem like the other solutions are, just a convention (and actually, it can be used with all 5 proposed options).

Reply:
Not sure what you mean how option 6 it can be used with all five solutions. It stands on its own, and the only viable approach as I read it. 

I'd also suggest that at this point in the game, after a few months of deliberations, we should focus on finalizing the decision and less on the doc structure and organization.

Reply:
> Not sure what you mean how option 6 it can be used with all five solutions. It stands on its own, and the only viable approach as I read it.

Because you can use the `VerX` suffix for non-stable methods regardless of which package the class the method is on lives in.

> I'd also suggest that at this point in the game, after a few months of deliberations, we should focus on finalizing the decision and less on the doc structure and organization.

This is not nitpicking about the structure of the document. This is about the fact that we still have decisions to make. If choosing option 6 still forces us to make a decision: "which package does the non-stable code live in", as it does currently with the way it's written, then we need to have the available options listed in section "Proposed changes", chapter 2, with the options that we rejected moved to Appendix C. Otherwise, we'll keep chasing our tail, as we did so many times on this issue before.

I want to have a clear choice: "These are the 2 options we're left with (with 4 options rejected, listed in appendix C), let's decide between the 2".

Also, notice that the various options are in a chapter titled "2. Separate "breakable" code from mono-CDK main modules". If option 6 does not allow shipping code that we can break, then it _definitely_ does not belong in that chapter.

Reply:
I'd like us to focus on the content and less on the structure at this point. 

As I read this, it's evident at this point that option 6 is the only viable approach to address our need for publishing non-final APIs. 

As I see it, the purpose of this PR is to make sure we didn't overlook anything before we sign off.

I agree that we need a few more details to cover the various types of APIs (modules, classes, methods, etc), but other then that the solution seems solid.

May I suggest that we leave the section reordering for a subsequent PR after this is finalized. I really rather not to have to review this once more if this is refactored again.

Reply:
> which package does the non-stable code live in

There is no more "unstable" code, all APIs are stable and they live in `aws-cdk-lib`. It is described in the release notes and in the advantages. Where else should it be mentioned?

Reply:
> Also, notice that the various options are in a chapter titled 2. Separate "breakable" code from mono-CDK main modules. If option 6 does not allow shipping code that we can break, then it definitely does not belong in that chapter.

I can move it under the chapter titled 1 "1. No more breaking changes in the main mono-CDK modules".

Reply:
To avoid adding a scary diff Im going to leave the section where it is to ease the review process, I will move it once we finalize the RFC